modules/bim/app/assets/stylesheets/bim/ifc_viewer/generic.sass

```diff
@@ -12,13 +12,12 @@
 
 // The IFC viewer shall fill the viewport to the bottom.
 // This is different to how we style #content else where.
-.controller-bim\/ifc_models\/ifc_models
+.controller-bim\/ifc_models\/ifc_viewer
   overflow: hidden
 
-  &.action-show,
-  &.action-defaults
+  &.action-show
     #content-wrapper
-      padding: 10px 20px 20px 20px
+      padding: 10px 0px 0px 10px
```

Comment:
That is a bit too much:
<img width="1360" alt="Bildschirmfoto 2020-03-03 um 09 22 26" src="https://user-images.githubusercontent.com/7457313/75756247-9fdb8a80-5d30-11ea-9f1d-52a070d41758.png">

I suggest to remove the special rule completely and use the spacing like on most other pages. This is currently already that way on the dev branch.